parameter_files/fates_params_default.cdl

```diff
@@ -1062,8 +1062,7 @@ data:
 
  fates_c2b = 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2 ;
 
- fates_cnp_eca_alpha_ptase = 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 
-    0.5, 0.5, 0.5 ;
+ fates_cnp_eca_alpha_ptase = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ;
```

Comment:
@rgknox Just double checking, was this change in defaults intended?

Reply:
yes.  This value is not used in the code, and the methods that are actually performed imply a value of 0.

Reply:
Cool, thanks for confirming!